migrations/2023-01-20-113235_add_attempt_id_to_payment_intent/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+ALTER TABLE PAYMENT_INTENT ADD COLUMN attempt_id VARCHAR(255) DEFAULT NULL;
+CREATE UNIQUE INDEX payment_attempt_attempt_id_index ON payment_attempt (attempt_id);
+DROP index payment_attempt_payment_id_merchant_id_index;
```

Comment:
Instead of dropping this index completely , should I maintain indexing on (payment_id, merchant_id) but it will not be unique. I guess this would help us in all payment_attempts related to an intent. 

Or should I create unique index on (payment_id, merchant_id, attempt_id). I'm not really sure which one would be efficient.

Reply:
We are querying based on (merchant_id, attempt_id) , (merchant_id, payment_id),  (merchant_id, connector_txn_id), 
so thinking of adding indexes on them. Let me know your thoughts.

Reply:
`(merchant_id, attempt_id)`-  unqiue index.
`(merchant_id, payment_id) (merchant_id, connector_txn_id)` - no unqiue index

Reply:
We should not keep the value as NULL

Reply:
The datatype of the `attempt_id` field should be same the` attempt_id` field in `payment_attempt`. VARCHAR(64).